depobs/website/do.py

```diff
@@ -18,6 +18,16 @@
 flasklog.setLevel(logging.ERROR)
 
 
+# override the summary logger from the Dockerflow class to add
+# logging of query string
+# https://github.com/mozilla-services/python-dockerflow/issues/44
+class Customflow(Dockerflow):
+    def summary_extra(self):
+        out = Dockerflow.summary_extra(self)
```

Comment:
Could replace with `out = super().summary_extra()` to feel more like modern python :)

Reply:
👍